pilot/pkg/proxy/envoy/externalservice.go

```diff
@@ -0,0 +1,183 @@
+// Copyright 2017 Istio Authors
```

Comment:
wasn't it supposed to be filenames with underscores?

Reply:
AFAIK, the Go convention is short names avoiding underscores except in specific cases:
1. Platform specific code: `linux_conn.go`
1. Test code: `config_test.go`

But this has been ignored elsewhere in the source code, so apparently we are not strict about it.

Reply:
Okay, I am not the go police. So, I hvae no strong opinions either way.

Reply:
@ijsnellf it's only now being clarified in our docs: https://github.com/istio/istio/pull/2915/files#diff-faf43fd5b538175ebab68d7d6af27ad8